doc/cascadia/profiles.schema.json

```diff
@@ -753,7 +753,16 @@
         },
         "backgroundImage": {
           "description": "Sets the file location of the image to draw over the window background.",
-          "type": ["string", "null"]
+          "oneOf": [
+            {
+              "type": ["string", null]
+            },
+            {
+              "enum": [
+                "desktopWallpaper"
+              ]
+            }
```

Comment:
Just double checking, you _don't_ need `"type": "string"` here? It looks like "backgroundImageAlignment" below has it. Either way, thank you!

Reply:
yeah i sure hope this works lol